RELEASES.md

```diff
@@ -1,3 +1,142 @@
+Version 1.54.0 (2021-07-29)
+============================
+
+Language
+-----------------------
+
+- [You can now use macros for values in key-value attributes.][83366]
```

Comment:
My understanding is that this is a little bit inaccurate. Any Rust expression is supported in key-value attributes now, but built-in macros only take strings (or macros that expand to strings). Procedural macros can accept other expressions in this position.